### DISCUSSION TITLE
Visual Studio C Emulator

### DISCUSSION THREAD

**Discussion by cvu1** [RudolphRiedel/FT800-FT813#133] [Q&A]
I see that the example from Bridgetek can complied into a PC Emulator. Is there any documents for setup using Visual Studio to create a EVE Emulator PC base? I try to make a simulator run on PC using C with Visual Studio.

Nots: nvm. I think I get the resolve.

**Comment by RudolphRrr** [comment]
I never tried this, for the most part I did my own library because I did not want to use the original FTDI solution.
However, I know that EVE Screen Editor comes with a "bt8xxemu.dll", so looking there might be a good starting point.

**Comment by hfieger** [comment]
Just open the generated project file and compile it.

**Reply by cvu1** [reply]
I mean I wan to create a simulator including touch/full functions on PC. Not just one screen from ESE.
